docs/insights/architecture.md

```diff
@@ -136,13 +87,13 @@ This sniff reports use of superfluous prefix or suffix "Exception" for exception
 
 This sniff checks the size of functions
 
-**Insight Class**: `ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff`
+**Insight Class**: `SlevomatCodingStandard\Sniffs\Files\FunctionLengthSniff`
```

Comment:
Here again, I would indicate InsightClass for v1 and InsightClass for v2 ? WDYT ?